lib/nostrum/consumer.ex

```diff
@@ -66,8 +66,13 @@ defmodule Nostrum.Consumer do
 
   @type channel_create :: {:CHANNEL_CREATE, {Channel.t()}, WSState.t()}
   @type channel_delete :: {:CHANNEL_DELETE, {Channel.t()}, WSState.t()}
+  @typedoc """
+  Dispatched when a channel is updated.
+  `old_channel` will be `nil` when the pre-update channel could not be fetched from the cache.
```

Comment:
Could you add a new line between the two sentences? ExDoc uses the first line to generate summaries, so we should try to keep it away from the rest of the provided info just for clarity.